test/specs/line.spec.js

```diff
@@ -409,6 +409,18 @@ define([
                     expect(actual).toBe(expected);
                 });
 
+                it('should provide lineCurve getter and setter', () => {
+                    let previous = lineChart.lineCurve(),
+                        expected = d3.curveBasis,
```

Comment:
Oops, I think this needs an update too.
Thanks

Reply:
Ah. Good catch